main.go

```diff
@@ -32,6 +32,8 @@ import (
 	"syscall"
 	"time"
 
+	"knative.dev/pkg/configmap"
```

Comment:
Why do we need this?

Reply:
This allows loading content from configmap in map[string]string form without implementing our own reader.

Reply:
Coulld we simple deserialize rules via yaml parser?